demos/checkboxradio/default.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Checkboxradio - Default functionality</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "input" ).checkboxradio();
+	});
+	</script>
+</head>
+<body>
+<div class="widget">
+	<h1>Checkbox and radio button widgets</h1>
+	<h2>Checkbox</h2>
+
+	<fieldset>
+		<legend>Rating</legend>
```

Comment:
Why would you use checkboxes for rating something? Adding multiple stars doesn't make much sense. Could switch these two examples, changing the legend below from "location" to "show timezones" or something where selecting multiple makes more sense.